integration/manifests/e2e-runner.yaml

```diff
@@ -21,7 +21,7 @@ spec:
             - tail
             - -f
             - /dev/null
-          image: kubeapps/integration-tests:v1.1.1
+          image: kubeapps/integration-tests:v2.0.0
```

Comment:
Small detail unrelated to this line, but to L8,L9,L13 and L17.
Given that the file has been renamed to `e2e-runner`, shouldn't we rename the deployment accordingly? `integration` seems too generic.

Reply:
Ah good idea, thanks!